datafusion/sqllogictest/test_files/array.slt

```diff
@@ -8072,6 +8072,18 @@ select [named_struct('a', 1, 'b', null)][-2];
 ----
 NULL
 
+statement ok
+COPY (select [[true, false], [false, true]] a, [false, true] b union select [[null, null]], null) to 'test_files/scratch/array/array_has/single_file.parquet' stored as parquet;
+
+statement ok
+CREATE EXTERNAL TABLE array_has STORED AS PARQUET location 'test_files/scratch/array/array_has/single_file.parquet';
+
+query B
+select array_contains(a, b) from array_has;
```

Comment:
This panics for me locally without the fix:

```
> select array_contains(a, b) from '/tmp/t.parquet';

thread 'main' panicked at /Users/andrewlamb/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/arrow-array-56.0.0/src/array/boolean_array.rs:91:13:
assertion `left == right` failed
  left: 2
 right: 1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
I recommend we also test with `array_contains (b, a)` as well to covert the other argument order